website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+
+### Launch Template
+
+Karpenter uses [AWS Bottlerocket OS](https://aws.amazon.com/bottlerocket/) by
+default. More specifically, Karpenter automatically creates a launch template
+with the name `Karpenter-<cluster name>-uuid` for each region where a node is
+provisioned.
+
+You can specify a different [launch
+template](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html),
+such as a customized version of Bottlerocket or a different platform
+altogether.
+
+**Template ID**
+- key: `node.k8s.aws/launch-template-id`
+- value example: `bottlerocket` (default)
+- value list: `aws ec2 describe-launch-templates`
+
+### Capacity Type (e.g., spot)
+
+- key: `node.k8s.aws/capacity-type`
+- values
+  - `on-demand` (default)
+  - `spot`
+
+Karpenter supports specifying capacity type and defaults to on-demand.
+
+Specify this value on the provisioner to enable spot instances. [Spot
+instances](https://aws.amazon.com/ec2/spot/) may be preempted, and should not
+be used for critical workloads.
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    node.k8s.aws/capacity-type: spot
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        node.k8s.aws/capacity-type: spot
+```
+
+### Architecture (e.g., ARM) 
+
+- key: `kubernetes.io/arch`
+- values
+  - `amd64` (default)
+  - `arm64`
+
+Karpenter supports `amd64` nodes, and `arm64` nodes. 
+
+**Example**
+
+*Set Default with provisioner.yaml*
+
+```yaml
+spec:
+  labels: 
+    kubernetes.io/arch: arm64
+```
+
+*Override with workload manifest (e.g., pod)*
+
+```yaml
+spec:
+  template:
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+```
+
+## Operating System
+
+- key: `kubernetes.io/os`
+- values
+  - `linux` (default)
+
+At this time, Karpenter only supports Linux OS nodes.
+
+### Availability Zones
+
+`topology.kubernetes.io/zone=us-east-1c`
+
+- key: `topology.kubernetes.io/zone`
+- value example: `us-east-1c` or `use1-az1`
+- value list: `aws ec2 describe-availability-zones --region <region-name>`
+
+Karpenter can be configured to create nodes in a particular zone. Karpenter
+supports (1) availability zone IDs, and (2) availability zone names. 
+
+Availability zone IDs, such as `use1-az1`, are consitent between AWS accounts.
+
+Availability zone names, such as `us-east-1c`, are randomly mapped to zone IDs
+on an account level. For example, the Availability Zone us-east-1a for your AWS
+account might not have the same location as us-east-1a for another AWS account. 
+
+[Learn more about Availablity Zone
+IDs.](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
+
+## Pod Labels
+
+### Accelerators, GPU 
+
+Accelerator (e.g., GPU) values include
+- `nvidia.com/gpu`
+- `amd.com/gpu`
+- `aws.amazon.com/neuron`
+
+Karpenter supports accelerators, such as GPUs. 
+
+To specify a specific GPU type, use the [instance type

```

Comment:
Can you clarify that this should be done in addition to `resources`. Otherwise the kubelet won't allocate the GPU to the pod.

Reply:
I clarified this, but the new content needs further review. I misunderstood this previously.